lib/std/crypto.zig

```diff
@@ -119,7 +119,10 @@ pub const nacl = struct {
 };
 
 const std = @import("std.zig");
-pub const randomBytes = std.os.getrandom;
+
+pub const random = struct {
+    pub const bytes = std.os.getrandom;
+};
```

Comment:
I think I didn't explain the idea well enough. It is to make `std.crypto.random` be a thread-local instance of the Random interface -  #6704